modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/legacy/xml_definition_serializer.js

```diff
@@ -36,7 +36,7 @@ AUI.add(
 		var serializeDefinition = function (xmlNamespace, metadata, json) {
 			var description = metadata.description;
 			var name = metadata.name;
-			var version = metadata.version;
+			var version = parseInt(metadata.version, 10);
```

Comment:
@MariMalta @MarinhoFeliphe Just a question, do we really need to convert this here? Thx

Reply:
I don't think we really need to, but in the xml right now the version is 1 and not 1.0, that's why the parseInt is there.

Reply:
I've tried with the 1.0 and doesn't work. Without the parseInt we receive an alert to enter a valid content.